arangodb-net-standard/CursorApi/CursorApiClient.cs

```diff
@@ -78,6 +73,7 @@ protected virtual WebHeaderCollection GetHeaderCollection(CursorHeaderProperties
         /// <param name="memoryLimit"></param>
         /// <param name="ttl"></param>
         /// <param name="transactionId">Optional. The stream transaction Id.</param>
+        /// <param name="headerProperties">Optional. Additional Header properties.</param>
         /// <param name="token">A CancellationToken to observe while waiting for the task to complete or to cancel the task.</param>
         /// <returns></returns>
         public virtual async Task<PostCursorResponse<T>> PostCursorAsync<T>(
```

Comment:
This overload is meant to be used as a convenience wrapper over the overload that takes body and header objects. I think we shouldn't add anymore parameters to it. If someone wants to use header parameters apart from `transactionId` then they should use the other overload:

```
PostCursorAsync<T>(
    PostCursorBody postCursorBody,
    CursorHeaderProperties headerProperties = null,
    CancellationToken token = default);
```

What do you think?

Reply:
Is reading from followers a common use case that it warrants making it part of the convenience method?